Jenkinsfile

```diff
@@ -278,17 +278,6 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
                     --channel ${params.STREAM} \
```

Comment:
Super minor: let's update that comment above to add e.g. "and makes the AMI public" ?

Reply:
Done in #110